Mathlib/Tactic/Set.lean

```diff
@@ -53,6 +47,13 @@ h2 : x = y
 -/
 ```
 -/
+-- This is called `setTactic` rather than `set`
+-- as we sometimes refer to `MonadStateOf.set` from inside `Mathlib.Tactic`.
+syntax (name := setTactic) "set" "!"? setArgsRest : tactic
+
+@[tactic_alt setTactic]
```

Comment:
Note to reviewers: the docstring got moved to the `syntax` declaration, and then this line was added.